digits/dataset/generic/test_views.py

```diff
@@ -1,13 +1,16 @@
 # Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
 from __future__ import absolute_import
 
-import json
-
 from bs4 import BeautifulSoup
+import json
+import numpy as np
```

Comment:
actually I am not sure I get the difference between a standard library import and a 3rd party import. Is it correct to say that `numpy` and `PIL.Image` are 3rd party imports and `json`, `os`, `tempfile` are standard library imports?